editoast/src/views/v2/timetable/stdcm.rs

```diff
@@ -196,8 +191,26 @@ async fn stdcm(
         );
     }
 
+    let max_run_time_result = get_max_run_time(
```

Comment:
Let's keep the same entire name everywhere?

```suggestion
    let maximum_run_time_result = get_max_run_time(
```